Gemfile

```diff
@@ -109,11 +109,10 @@ group :development, :test do
   gem 'net-http-persistent'
   gem 'rinku'
   gem 'rspec', require: false
-  gem 'selenium-webdriver', '~> 4.0'
+  gem 'selenium-webdriver', '~> 4.25'
```

Comment:
Is 4.25 required here? It seems to work fine for me locally on 4.23

If we don't have a specific requirement on 4.25, I'd prefer to update the specification in the `Gemfile` to `~> 4.6` and add a comment explaining that we have a dependency on Selenium Manager. We can still update the actual installed version in `Gemfile.lock` to whatever the latest is, of course, but the Gemfile should reflect specific intent.